code/__DEFINES/time.dm

```diff
@@ -50,3 +50,83 @@ When using time2text(), please use "DDD" to find the weekday. Refrain from using
 #define DS2TICKS(DS) ((DS)/world.tick_lag)
 
 #define TICKS2DS(T) ((T) TICKS)
+
+/*Timezones*/
+
+/// Line Islands Time
+#define TIMEZONE_LINT 14
+
+/// Tokelau Time
+#define TIMEZONE_TKT 13
+
+/// Tonga Time
+#define TIMEZONE_TOT 13
+
+/// New Zealand Daylight Time
+#define TIMEZONE_NZDT 13
+
+/// New Zealand Standard Time
+#define TIMEZONE_NZST 12
+
+/// Norfolk Time
+#define TIMEZONE_NFT 11
+
+/// Lord Howe Standard Time
+#define TIMEZONE_LHST 10.5
+
+/// Australian Eastern Standard Time
+#define TIMEZONE_AEST 10
+
+/// Australian Central Standard Time
+#define TIMEZONE_ACST 9.5
+
+/// Australian Central Western Standard Time
+#define TIMEZONE_ACWST 8.75
+
+/// Australian Western Standard Time
+#define TIMEZONE_AWST 8
+
+/// Christmas Island Time
+#define TIMEZONE_CXT 7
+
+/// Cocos Islands Time
+#define TIMEZONE_CCT 6.5
+
+/// Central European Summer Time
+#define TIMEZONE_CEST 2
+
+/// Coordinated Universal Time
+#define TIMEZONE_UTC 0
+
+/// Eastern Daylight Time
+#define TIMEZONE_EDT -4
+
+/// Central Daylight Time
+#define TIMEZONE_CDT -5
+
+/// Mountain Daylight Time
+#define TIMEZONE_MDT -6
+
+/// Mountain Standard Time
+#define TIMEZONE_MST -7
+
+/// Pacific Daylight Time
+#define TIMEZONE_PDT -7
+
+/// Alaska Daylight Time
+#define TIMEZONE_AKDT -8
+
+/// Hawaii-Aleutian Daylight Time
+#define TIMEZONE_HDT -9
+
+/// Hawaii Standard Time
+#define TIMEZONE_HST -10
+
+/// Cook Island Time
+#define TIMEZONE_CKT -10
+
+/// Niue Time
+#define TIMEZONE_NUT -11
+
+/// Anywhere on Earth
+#define TIMEZONE_AoE -12
```

Comment:
```suggestion
#define TIMEZONE_ANYWHERE_ON_EARTH -12
```

Nitpick, but the abbreviation doesn't really help us here (especially since AoE is most commonly associated with "area of effect")

Reply:
Also, macros should be SCREAMING_SNAKE_CASE most of the time for consistency.